packages/slate-react/src/plugins/core.js

```diff
@@ -82,8 +81,8 @@ function Plugin(options = {}) {
     // the selection has gotten out of sync, and adjust it if so. (03/18/2017)
     const window = getWindow(e.target)
     const native = window.getSelection()
-    const a = getPoint(native.anchorNode, native.anchorOffset, state, editor)
-    const f = getPoint(native.focusNode, native.focusOffset, state, editor)
+    const a = findPoint(native.anchorNode, native.anchorOffset, state)
+    const f = findPoint(native.focusNode, native.focusOffset, state)
```

Comment:
Actually any place where we can doing `findPoint` for both anchor and focus can be slightly improved by checking if `collapsed` and if so only doing it once. But I'm not sure this is a bottleneck, so leaving for later.